manifests/latest/vsphere-7.0/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -150,7 +150,7 @@ metadata:
   name: csi-feature-states
   namespace: kube-system
 ---
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
for 1.19+ we see

> Warning: storage.k8s.io/v1beta1 CSIDriver is deprecated in v1.19+, unavailable in v1.22+; use storage.k8s.io/v1 CSIDriver

Reply:
So, if user deploy this yaml in 1.18 or lower, we still need v1beta1 ?

Reply:
Also, do we need k8s version specific yamls ?

Reply:
AFAIK, vSphere CSI migration is only supported in k8s 1.19 and above. In that case, we can use `storage.k8s.io/v1 CSIDriver`